src/interfaces/IApolloCoordinatorV2.sol

```diff
@@ -12,6 +11,6 @@ interface IApolloCoordinatorV2 {
     function requestDataFeed(string memory dataFeedId, uint256 callbackGasLimit) external;
 
     event PriceFeedRequested(
-        string indexed dataFeedId, uint256 callbackGasLimit, address indexed requester
+        string dataFeedId, uint256 callbackGasLimit, address indexed requester
```

Comment:
why removed `indexed` here?

Reply:
Indexed string results in abi-encoded value in output. Meaning if this string was indexed, in the event logs we would see not `"BTC/USD"`, but keccak256("BTC/USD") = `"0xee62665949c883f9e0f6f002eac32e00bd59dfe6c34e92a91c37d6a8322d6489"`